.github/workflows/test.yml

```diff
@@ -242,6 +265,9 @@ jobs:
       fail-fast: false
       matrix:
         include: ${{ fromJson(needs.discover.outputs.groups) }}
+    env:
+      LOTUS_RUN_EXPENSIVE_TESTS: 1
```

Comment:
This is set by default because we `CI` env var is set, but I like that this makes it more explicit.

Reply:
Yup, that's the first item in the PR description!